libsolidity/parsing/Parser.cpp

```diff
@@ -678,6 +678,7 @@ ASTPointer<VariableDeclaration> Parser::parseVariableDeclaration(
 	ASTNodeFactory nodeFactory = _lookAheadArrayType ?
 		ASTNodeFactory(*this, _lookAheadArrayType) : ASTNodeFactory(*this);
 	ASTPointer<TypeName> type;
+	ASTPointer<StructuredDocumentation> documentation = parseStructuredDocumentation();
```

Comment:
```suggestion
	ASTPointer<StructuredDocumentation> const documentation = parseStructuredDocumentation();
```
I'd suggest to make things const that are not meant to  be modified. (It's not part of our coding style guidelines, however I believe it makes things clearer to the reader looking at unknown / foreign code). What do you think?